test/disk_size_test.rb

```diff
@@ -285,7 +305,15 @@
       disk_size = Y2Storage::DiskSize.MiB(12) / 3
       expect(disk_size.to_i).to be == 12 / 3 * 1024**2
     end
-    it "should refuse another type" do
+    it "should accept division by a float" do
```

Comment:
I think we are overspecifying. What should this operation mean anyway?

Because then we could also say that strings make sense, just like for `+` and `-`
It's complicated. So I think we should not commit to more than we need.